docs/integrations/sources/surveymonkey.md

```diff
@@ -1,72 +1,103 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 # SurveyMonkey
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
+This guide contains instructions for setting up an Airbyte Cloud or Open Source connector for SurveyMonkey.
 
 :::note
 
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
+OAuth for Survey Monkey is officially supported only for the US. Airbyte is testing how to enable it in the EU. If you run into issues, [reach out to support](mailto:product@airbyte.io).
 
 :::
 
+## Prerequisites
+
+<Tabs groupId="platform">
+<TabItem value="oauth" label="Airbyte Cloud" default>
+
+<!-- env:cloud -->
+
+You have an [Airbyte Cloud](https://docs.airbyte.com/using-airbyte/getting-started/#sign-up-for-airbyte-clouds) account.
+
+
+<!-- /env:cloud -->
+
+</TabItem>
+
+<TabItem value="http-basic" label="Airbyte Open Source">
+
 <!-- env:oss -->
 
-## Prerequisites
+You have:
 
-**For Airbyte Open Source:**
+- Installed [Airbyte](https://docs.airbyte.com/deploying-airbyte/quickstart).
```

Comment:
I spent some time finding this doc and setting it up, so I added it as a prereq with a link to the doc.